### ISSUE TITLE
Remove Nano defender from default collections list, as it is malware now.

### ISSUE THREAD

**Issue by aquaspy** [fork-maintainers/iceraven-browser#225]
https://www.reddit.com/r/firefox/comments/jbua53/nanoadblocker_nanodefender_is_malware_now/

### Who will benefit from it?
User that dont know about it

**Comment by hbarsaiyan** [comment]
It's only the chromium versions that are affected by this. The firefox version is not affected by the ownership change as it is maintained by a different person (LiCybora) and he will not make the changes done by the new developers. The future of the Firefox fork is undecided atm.

```
NA and ND with LiCybora as author on AMO or on my GitHub repository are still under my control and independent from any entities or people.
```

You can read the whole thing [here](https://github.com/LiCybora/NanoDefenderFirefox/issues/187)

tl;dr: Your data is safe on the Firefox version.

**Comment by interfect** [comment]
Sounds like everything is fine for us for now, then.
